### PR TITLE
[AC-2928] Create automatic app login policy

### PR DESCRIPTION
**Pull request by kspearrin**
## 🎟️ Tracking

https://bitwarden.atlassian.net/browse/AC-2928

## 📔 Objective

Add a new policy to allow automatic app login functionality from a IDP app dashboard.

## 📸 Screenshots

image

image

## ⏰ Reminders before review

- Contributor guidelines followed
- All formatters and local linters executed and passed
- Written new unit and / or integration tests where applicable
- Protected functional changes with optionality (feature flags)
- Used internationalization (i18n) for all UI strings
- CI builds passed
- Communicated to DevOps any deployment requirements
- Updated any necessary documentation (Confluence, contributing docs) or informed the documentation team

## 🦮 Reviewer guidelines

- 👍 (`:+1:`) or similar for great changes
- 📝 (`:memo:`) or ℹ️ (`:information_source:`) for notes or general info
- ❓ (`:question:`) for questions
- 🤔 (`:thinking:`) or 💭 (`:thought_balloon:`) for more open inquiry that's not quite a confirmed issue and could potentially benefit from discussion
- 🎨 (`:art:`) for suggestions / improvements
- ❌ (`:x:`) or ⚠️ (`:warning:`) for more significant problems or concerns needing attention
- 🌱 (`:seedling:`) or ♻️ (`:recycle:`) for future improvements or indications of technical debt
- ⛏ (`:pick:`) for minor or nitpick changes

### CODE REVIEW

File: apps/web/src/locales/en/messages.json [hunk 1]
```diff
@@ -5276,6 +5276,18 @@
   "updateWeakMasterPasswordWarning": {
     "message": "Your master password does not meet one or more of your organization policies. In order to access the vault, you must update your master password now. Proceeding will log you out of your current session, requiring you to log back in. Active sessions on other devices may continue to remain active for up to one hour."
   },
+  "automaticAppLogin": {
+    "message": "Automatic log in for applications"
+  },
+  "automaticAppLoginDesc": {
+    "message": "Allow automatic login for members. Login forms will automatically be filled and submitted for apps launched from your configured identity provider."
```
Comment (abergs) [inline apps/web/src/locales/en/messages.json:5283]: @kspearrin Do we want to be more expressive about the integration with IDPs and the flow that this enables?

Comment (kspearrin) [reply apps/web/src/locales/en/messages.json:5283]: Yes, but I am not sure this is the place for it. We will have to expand on that more in documentation.

Comment (abergs) [reply apps/web/src/locales/en/messages.json:5283]: @kspearrin Gotcha

**Review by cagonzalezcs — APPROVED** [review]
The work that affects changes to the files owned by the autofill team was previously approved by @jprusik . No other changes have been made to those files since then, as a result I'm going to provide the autofill team approval for this.

**Review by r-tome — APPROVED** [review]

**Pull request merged**